proposal for how the team operates (#215)
rust-lang/rust-project-goals: src/admin/author_updates.md

# Author updates

## Soliciting updates

Triagebot can ping project-goal owners for updates. To use it, go to Zulip and execute a command like this.

```
@triagebot ping-goals 14 Oct-21
```

The first number (14) is a threshold, it is typically set to the current day of the month (e.g., the above command assumes it is Oct 14). It means "if they have posted a comment in the last 14 days, don't bug them". 

The second string ("Oct-21") is the deadline for updates to be included.

We need to improve this UI.

## Drafting the post

> The `cargo rpg` tool offers the `updates` command ([documented here](./updates.md)) which will prepare a rough draft blog post using the [updates template][]. The rough draft is prepared using an LLM which will require setting up an AWS account.

Reviewer: ```suggestion
The `cargo rpg` tool offers the `updates` command ([documented here](./updates.md)) which will prepare a rough draft blog post using the [updates template][]. @nikomatsakis currently prepares the rough draft using an LLM via an AWS account.
```

Still not happy that this is being used; I think we should just concatenate the provided updates and use *that* as the starting rough draft. In the spirit of *documenting* the current process before changing it, :+1: for at least documenting this.

Reviewer 2: Why is it used?

Was this technology selected based on meeting a criteria, or based on vibes?

Author: ```suggestion
The [blog post template][updates template] includes a longer description of the flagship goals as well as a bullet-point or two for each of the other goals -- more than that and the blog post becomes overwhelming, the goal is mostly to let people find the goal they're interested in and click to read the full details. You can fill out the template by hand but you can also use the `cargo rpg updates` ([documented here](./updates.md)) to prepare a rough draft; the tool uses an LLM currently hosted on AWS Bedrock and hence requires setting up an AWS account.
```

This includes both the reasoning behind the tool and the documentation. As the update says, if someone prefers to prepare the first draft by hand, I wouldn't stop you, I'm sure you'd do a better job than my quickly crafted script.
